### PR TITLE
netclient-userspace-docker workflow

### PR DESCRIPTION
**Pull request by mattkasun**

### CODE REVIEW

**Comment by theguy951357** [comment]
according to the kitemaker ticket, testing cannot be done until merged into master. I will test when that happens

File: docker/Dockerfile-netclient-multiarch-userspace [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+ENV GO111MODULE=auto
+RUN apk add --update git build-base libmnl-dev iptables libpcap-dev
+COPY . .
+
+RUN GOOS=linux CGO_ENABLED=1 /usr/local/go/bin/go build -tags headless -ldflags="-w -s" -o netclient-app .
+
+WORKDIR /root/
+
+RUN git clone https://git.zx2c4.com/wireguard-go && \
```
Comment (0xdcarns) [inline docker/Dockerfile-netclient-multiarch-userspace:14]: we should not need wireguard-tools or wg-go in the userspace implementation

**Review by 0xdcarns — CHANGES_REQUESTED** [review]

File: docker/Dockerfile-netclient-multiarch-userspace (hunk 1 above)
Comment (mattkasun) [reply docker/Dockerfile-netclient-multiarch-userspace]: removed

File: docker/Dockerfile-netclient-multiarch-userspace [hunk 2]
```diff
@@ -9,26 +9,11 @@ COPY . .
...
-    make && \
-    make install
-
 FROM alpine:3.16.2
 
 WORKDIR /root/
 
-RUN apk add --no-cache --update bash libmnl gcompat iptables openresolv iproute2
-COPY --from=builder /usr/bin/wireguard-go /usr/bin/wg* /usr/bin/
+RUN apk add --no-cache --update bash libmnl gcompat iptables openresolv iproute2 libpcap
```
Comment (0xdcarns) [inline docker/Dockerfile-netclient-multiarch-userspace:16]: remove libpcap as well

**Review by 0xdcarns — CHANGES_REQUESTED** [review]

**Pull request closed without merging**